pkg/sql/pgwire/testdata/pgtest/int2vector

```diff
@@ -0,0 +1,29 @@
+send
+Query {"String": "DROP TABLE IF EXISTS t"}
+Query {"String": "CREATE TABLE t (a int primary key, b text)"}
+----
+
+until
+ReadyForQuery
+ReadyForQuery
+----
+{"Type":"CommandComplete","CommandTag":"DROP TABLE"}
+{"Type":"ReadyForQuery","TxStatus":"I"}
+{"Type":"CommandComplete","CommandTag":"CREATE TABLE"}
+{"Type":"ReadyForQuery","TxStatus":"I"}
+
+send
+Parse {"Query": "select i.indoption from pg_index i join pg_class c on i.indrelid = c.oid where c.relname = 't';"}
+Bind {"ParameterFormatCodes": [0], "ResultFormatCodes": [1]}
+Execute
+Sync
+----
+
+until
+ReadyForQuery
```

Comment:
could you add a comment that says the test is checking that the size of the binary data is returned as 2 byte ints, and maybe include a link to #111911 ?